ts/components/services/ServiceListItem.tsx

```diff
@@ -90,7 +130,17 @@ export class ServiceListItem extends React.PureComponent<Props> {
           </Grid>
         </Left>
         <Right>
-          <IconFont name="io-right" color={variables.brandPrimary} />
+          {isLongPressModeEnabled ? (
+            <Col size={2}>
```

Comment:
Are you sure you need **_<Col>_**?

Reply:
According to me yes because I need to decide what icon to show. What doesn't convince you?

Reply:
Sorry, maybe it should be not clear what I'm referring to.
I think **_<Col size{2}>_** could be removed. Since it is a _Right_ element maybe you no need it

Reply:
Sorry, i changed this. I used it seeing it in detail but actually it is not needed here